Cookbook/Current.md

```diff
@@ -0,0 +1,18 @@
+# How to use `Current` in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+It's purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### do
+- access `Current` **only** from `Builder`s layer;
+- pass the required dependencies down the chain;
+- contribute to it;
+
+#### don't
+- access `Current` from any other layer other than `Builder`. I.E all the other layers (ViewModel, Flowcontroller, ViewController, Model, BusinessController) continue to receive its dependencies throught injection, preferably via `init`;
+- inject `current` as a dependency;
+- try to mutate `Current`. In `Release` configuration it's a constant `let`;
```

Comment:
Could this be further explained? For example I didn't really follow the release issue on slack, so I have no idea why the Release config has to be a constant property.

Reply:
it's basically this bit: 
```swift
#if DEBUG
public var Current: World = .production
#else
public let Current: World = .production
#endif
```

We can mutate for Debug and specially for Tests. But production not at all. 
But it mutates via `setPropertyName` which changes the value of `MutableProperties` which maintains the immutability of `Current`. 
Semantics ✨ 

I'll think how that can be explained in there.
👍

Reply:
just to highlight

`public var`
`public let` 

I took time to realise the difference of these 2 lines. lol

Reply:
added the snippet